apps/backend/src/modules/onboarding/libs/types/types.ts

```diff
@@ -0,0 +1,7 @@
+export { type UserDto } from "~/modules/users/users.js";
```

Comment:
Shouldn't be exported from here. Import directly from users module where you need it